CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## UNRELEASED
+
+### Dependencies
+
+- Switch rocksdb binding from gorocksdb to grocksdb, update librocksdb dependency to `v7.10.2`, [#42](https://github.com/cometbft/cometbft-db/pull/42).
+
```

Comment:
Please see the contributing guidelines for how to add a changelog entry: https://github.com/cometbft/cometbft/blob/main/CONTRIBUTING.md#changelog

It needs to exist as a separate file in the `.changelog/unreleased/dependencies` directory.

Reply:
done